### PR TITLE
Update/Fix some things

### PR DESCRIPTION
**Pull request by NoBody-IDK**
🐒

### CODE REVIEW

**Review by PabloRNC — DISMISSED** [review]
For a better sintaxt you can write: Moreover, categories can be created within a folder and then put the command into it.

File: src/content/docs/commands/subcommands.mdx [hunk 1]
```diff
@@ -26,6 +26,29 @@ It's suppose you have the following directory structure:
...
+import { type CommandContext, Declare, Command, Options } from "seyfert";
+import { AddCommand } from "./create.command";
+import { DeleteCommand } from "./delete.command";
+
+@Declare({
+	name: "account",
+	description: "account command"
+})
+@Options([AddCommand, DeleteCommand]) // If the sub-commands are in the same directory can use  @Autoload()
+export default class AccountCommand extends Command {}
```
Comment (socram03) [inline src/content/docs/commands/subcommands.mdx]:
```suggestion
})
// Being in the same folder with @AutoLoad() you can save this
@Options([AddCommand, DeleteCommand])
export default class AccountCommand extends Command {}
```

File: src/content/docs/guides/first-command.mdx [hunk 2]
```diff
@@ -64,6 +64,7 @@ import {
   createBooleanOption,
   type CommandContext
 } from 'seyfert';
+import { MessageFlags } from 'seyfert/lib/common';
```
Comment (socram03) [inline src/content/docs/guides/first-command.mdx]: Adding this breaks highlight formatting, moves markdown values

File: src/content/docs/guides/first-command.mdx [hunk 3]
```diff
@@ -98,7 +99,7 @@ export default class PingCommand extends Command {
 <FileTree>
 - src
 	- commands
-		- **ping.ts**
+		- **ping.ts** // Also, you can create a folder for categories and put the command there.
```
Comment (socram03) [inline src/content/docs/guides/first-command.mdx]: Move the comment up, at the end of the line it is barely visible

**Review by socram03 — CHANGES_REQUESTED** [review]

**Review by socram03 — DISMISSED** [review]

**Review by FreeAoi — COMMENTED** [review]
HI! could you revert this on try to fetch contributors by fetch api pls?

File: src/content/docs/commands/subcommands.mdx [hunk 4]
```diff
@@ -35,11 +59,11 @@ import { type CommandContext, Declare, SubCommand } from "seyfert";
 	name: "create",
 	description: "create a new something"
 })
-export class AddCommand extends SubCommand {
+export class CreateCommand extends SubCommand {
```
Comment (FreeAoi) [inline src/content/docs/commands/subcommands.mdx:63]: When using autoload you must export default the SubCommand class!!

**Review by FreeAoi — APPROVED** [review]

**Review by MARCROCK22 — APPROVED** [review]

**Pull request merged**